src-ghc/Pact/Bench.hs

```diff
@@ -86,7 +89,10 @@ benchNFIO bname = bench bname . nfIO
 runPactExec :: PactDbEnv e -> RefStore -> ParsedCode -> IO Value
 runPactExec dbEnv refStore pc = do
   t <- Transactional . fromIntegral <$> getCPUTime
-  toJSON . _erOutput <$> evalExec (setupEvalEnv dbEnv entity t (initMsgData (initialHashTx H.Blake2b_512)) refStore freeGasEnv permissiveNamespacePolicy) pc
+  toJSON . _erOutput <$>
+    evalExec (setupEvalEnv dbEnv entity t (initMsgData (initialHashTx H.Blake2b_512))
+              refStore freeGasEnv permissiveNamespacePolicy noSPVSupport)
+    pc
 
 benchKeySet :: KeySet
 benchKeySet = KeySet [PublicKey "benchadmin"] (Name ">" def)
```

Comment:
These `pc`'s are a small legibility wibble. Maybe we should `where` on the `evalExec` so those `<$>`'s aren't line breaking on their arguments?

Reply:
Turns out they can go on the same line
```suggestion
toJSON . _erOutput <$>
    evalExec (setupEvalEnv dbEnv entity t (initMsgData (initialHashTx H.Blake2b_512))
              refStore freeGasEnv permissiveNamespacePolicy noSPVSupport) pc
```

Reply:
done